stable/phpmyadmin/README.md

```diff
@@ -0,0 +1,80 @@
+# PhpMyAdmin
+
+[Phpmyadmin](https://www.phpmyadmin.net/) is a free and open source administration tool for MySQL and MariaDB. As a portable web application written primarily in PHP, it has become one of the most popular MySQL administration tools, especially for web hosting services.
+
+## TL;DR;
```

Comment:
-> `TL;DR`

Reply:
seems like additional ; is largely used

https://github.com/kubernetes/charts/search?utf8=%E2%9C%93&q=tl%3Bdr%3B&type=
https://github.com/kubernetes/charts/tree/master/stable/wordpress#tldr

If this is a blocker to land it I am happy to change it.